mappings/net/minecraft/client/player/EntityPlayerClient.mapping

```diff
@@ -5,6 +5,7 @@ CLASS none/boe net/minecraft/client/player/EntityPlayerClient
 	METHOD a swingHand (Lnone/qz;)V
 		ARG 0 hand
 	METHOD c getBlockPos ()Lnone/cn;
+	METHOD c setCurrentHand (Lnone/qz;)V
```

Comment:
Agreed, if it's the client hand, the name should reflect that as to prevent confusion and incorrect side errors.